deploy/src/main/config/hono-adapter-mqtt-vertx-config.yml

```diff
@@ -43,3 +43,6 @@ hono:
     keyPath: ${secret.path}/mqtt-adapter-key.pem
     certPath: ${secret.path}/mqtt-adapter-cert.pem
     trustStorePath: ${secret.path}/trusted-certs.pem
+  plan:
+    prometheusBased:
+      queryURL: http://prometheus-operated.hono:9090/api/v1/query
```

Comment:
if this is a property for a Prometheus specific implementation then FMPOV the only thing that needs to be defined is the Prometheus host name, isn't it? The relative URI of the query endpoint should not need to be configured, should it?

Reply:
@sophokles73 For example it can also be deployed or mapped to a different port. In order to give more flexibility, I have made the entire url configurable.

Reply:
are you going to change this? at least, do not *require* users to provide the resource path, i.e. the `/api/v1/query`, by e.g. checking if the provided URI contains a resource path and, if not, append this default.
Please also rename the property to *queryUrl* so that it complies with our method naming strategy.

Reply:
@sophokles73 I have made it to support two properties namely _host_ and _port_.